compiler/fir/resolve/src/org/jetbrains/kotlin/fir/resolve/transformers/FirSupertypesResolution.kt

```diff
@@ -284,21 +285,24 @@ private class FirSupertypeResolverVisitor(
         return resolveSpecificClassLikeSupertypes(classLikeDeclaration) { transformer, scope ->
             ArrayList(supertypeRefs).mapTo(mutableListOf()) {
```

Comment:
Not related to this change. But isn't this copy redundant?

Reply:
Indeed, good catch!